source/frontend/src/components/propertySelector/MapClickMonitor.tsx

```diff
@@ -1,36 +1,36 @@
 import * as React from 'react';
-import { useContext } from 'react';
 
-import { MapStateContext } from '@/components/maps/providers/MapStateContext';
+import { useMapStateMachine } from '@/components/common/mapFSM/MapStateMachineContext';
 import useDraftMarkerSynchronizer from '@/hooks/useDraftMarkerSynchronizer';
 import { usePrevious } from '@/hooks/usePrevious';
 import useDeepCompareEffect from '@/hooks/util/useDeepCompareEffect';
-import { mapFeatureToProperty } from '@/utils/mapPropertyUtils';
+import { featuresetToMapProperty } from '@/utils/mapPropertyUtils';
 
 import { IMapProperty } from './models';
 
 interface IMapClickMonitorProps {
   addProperty: (property: IMapProperty) => void;
-  modifiedProperties: IMapProperty[];
+  modifiedProperties: IMapProperty[]; // TODO: this should be just a list of lat longs
```

Comment:
should it? doesn't this feed into the actually selected properties list?

Reply:
The modifiedProperties is only used to synchronize the markers on the map. useDraftMarkerSynchronizer only uses the locations from the passed properties.